doc/tutorials/04_agent_versus_non_agent_regret.ipynb

```diff
@@ -0,0 +1,705 @@
+{
```

Comment:
<p>At the moment, the tutorial that explains starting points is in the <code>advanced_tutorials</code> folder, if this assumes the reader has that knowledge already perhaps we need to reorder, or simply put this tutorial in the advanced folder and link to the other one here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gambitproject/gambit/pull/642/discussion/#comment-2630560115'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>